app/lib/config/config.go

```diff
@@ -128,25 +128,34 @@ type ServerToml struct {
 	Description string
 }
 
-// ReadGroupToml reads a group.toml file and returns the list of Entity
-// described in the file.
-func ReadGroupToml(f io.Reader) (*sda.EntityList, error) {
+// ReadGroupDescToml reads a group.toml file and returns the list of Entities
+// and descriptions in the file.
+func ReadGroupDescToml(f io.Reader) (*sda.EntityList, []string, error) {
```

Comment:
Should we create a `Server` & `Group` struct that contains the decoded data instead of passing each data differently ?
Not sure, but if we ever add a new field to a `Server` (like certificate when we'll do TLS) then we have to return again an additional return argument...